fastlane/jetpack_metadata/fr-FR/description.txt

```diff
@@ -1,9 +1,47 @@
-Bénéficiez d’outils de sécurité et de performance puissants avec Jetpack pour iOS.
+Jetpack pour WordPress
```

Comment:
For reference, matches changes that were announced in pcdRpT-z4-p2 and that landed earlier this sprint in https://github.com/wordpress-mobile/WordPress-iOS/pull/19083 👍